insights_configuration.rst

```diff
@@ -62,10 +66,11 @@ Download Insights Core RPM
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 Download the last stable version of the Insights core::
 
-    curl https://api.access.redhat.com/r/insights/v1/static/core/insights-core.egg.asc > last_stable.egg.asc
-    sudo mv last_stable.egg.asc /var/lib/insights/last_stable.egg.asc
-    curl https://api.access.redhat.com/r/insights/v1/static/core/insights-core.egg > last_stable.egg
-    sudo mv last_stable.egg /var/lib/insights/last_stable.egg
+sudo sh lay-the-eggs-osx.sh
```

Comment:
I think this needs to be tabbed over to be shown as code in the rst.

Reply:
Fixed.  Thanks.